src/detect-engine-analyzer.c

```diff
@@ -861,6 +863,22 @@ static void DumpMatches(RuleAnalyzer *ctx, JsonBuilder *js, const SigMatchData *
                 jb_close(js);
                 break;
             }
+            case DETECT_TCPMSS: {
+                const DetectU16Data *cd = (const DetectU16Data *)smd->ctx;
+
+                jb_open_object(js, "tcp_mss");
+                const char *flag = DetectTcpmssModeToString(cd->mode);
+                jb_set_string(js, "operand", flag);
+                if (strcmp(flag, "range") == 0) {
+                    jb_set_uint(js, "min", cd->arg1);
+                    jb_set_uint(js, "max", cd->arg2);
```

Comment:
@jasonish we need to settle on what to use here for single values and ranges. Any thoughts on what makes sense?

Reply:
Minimally, with sime inspiration from the Elasticsearch query language:
```json
		{
		    "name": "tcp.mss",
		    "equals": 5
		},
		{
		    "name": "tcp.mss",
		    "range": {
			"min": 5,
			"max": 10
		    }
		}
		{
		    "name": "tcp.mss",
		    "gt": 5
		}
```

But this breaks the format, is there a reason why we use "name", then use a sub-object other than we are already doing it? To keep with the current format I think this looks good:
```json
		{
		    "name": "tcp.mss",
		    "tcp_mss": {
			"equals": 5
		    }
		},
		{
		    "name": "tcp.mss",
		    "tcp_mss": {
			"range": {
			    "min": 5,
			    "max": 10,
			}
		    }
		},
		{
		    "name": "tcp.mss",
		    "tcp_mss": {
                        "operator": "gt",
			"gt": 5
		    }
		},
```
Note that I added the `operator` to the last one, but I don't think its really needed.

Reply:
I'm good with Jason's last suggestion here.